extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -117,6 +117,7 @@ const TEMPLATE = {
           tag: 'div',
           attrs: dict({
             'role': 'button',
+            'aria-label': 'Info',
```

Comment:
Should we be a bit more descriptive about what clicking this button would do?
For a non visually impaired user, recognizing the Story format and seeing the "audio" button makes it very obvious about what's going to happen if the button is clicked.
We might want to improve the `aria-label` into something like "Story information", "Unmute the story content" or "Open/close the menu"

Wdyt?

Reply:
The current text could be improved, I'll edit these to make them slightly more informative, but it's definitely a goal to keep aria-labels concise in conveying their purpose. I'll change the text and change to using i18n.